server/src/core/server/app/handlers/api/tenor/index.ts

```diff
@@ -133,6 +93,7 @@ export const tenorSearchHandler =
     url.searchParams.set("key", apiKey);
     url.searchParams.set("limit", `${SEARCH_LIMIT}`);
     url.searchParams.set("contentfilter", contentFilter);
+    url.searchParams.set("media_filter", "gif,nanogif");
```

Comment:
I added this `media_filter` to only return the response object formats that we need when we are calling the Tenor api here.